.github/workflows/build.yaml

```diff
@@ -92,8 +93,8 @@ jobs:
             echo "Python location: $PYTHON"
           fi
 
-          autobuild configure -- -DVIEWER_CHANNEL="Second Life Test ${GIT_REF##*/}"
-          autobuild  build --no-configure
+          autobuild configure --id "${{ github.run_id }}" -- -DVIEWER_CHANNEL="Second Life Test ${GIT_REF##*/}"
```

Comment:
Hmm, I'm no longer sure `github.run_id` is the right variable. Consider https://github.com/secondlife/viewer/actions/runs/5205525429. What I think we want is version 6.6.13.5205525429 so we could just paste that last component into the .../viewer/actions/runs/ URL. Instead the viewer versions are 231482258 (64-bit viewers) or 231582302 (32-bit viewer).

Reply:
Yeah you're right that's not what I expected from reading the docs.  I'll keep investigating this

Reply:
correction.  github.run_id is properly matching with that URL component as I read it.  for example: https://github.com/secondlife/viewer/actions/runs/5384282553 

even though those builds are currently failing due to the other issues described in this PR, they do properly set the build to be 6.6.13.5384282553 for all 3 builds.

Reply:
I do see that in your most recent branch build, https://github.com/secondlife/viewer/actions/runs/5559609923 in fact produces viewer version 6.6.13.5559609923, which is what I was hoping for. Not sure what went wrong with the build I first cited above.

Reply:
it looks like just a minor confusion.  double checking now, the change 3ffa57473d4aec6a32807873513f7d880ec4bc67 that it was built from preceded this branch by a couple commits